roles/etcd/tasks/join_etcd-events_member.yml

```diff
@@ -1,11 +1,12 @@
 ---
 - name: Join Member | Add member to etcd-events cluster
-  shell: "{{ bin_dir }}/etcdctl --endpoints={{ etcd_events_access_addresses }} member add {{ etcd_member_name }} {{ etcd_events_peer_url }}"
+  shell: "{{ bin_dir }}/etcdctl member add {{ etcd_member_name }} {{ etcd_events_peer_url }}"
   register: member_add_result
   until: member_add_result.rc == 0
   retries: "{{ etcd_retries }}"
   delay: "{{ retry_stagger | random + 3 }}"
   environment:
+    ETCDCTL_ENDPOINTS: "{{ etcd_events_access_addresses }}"
```

Comment:
Why didn't you set `ETCDCTL_API` here ?

Reply:
Missed it, adding that in latest commit.